src/app/workflow/dag/wdl-viewer/wdl-viewer.component.scss

```diff
@@ -0,0 +1,22 @@
+@import "~pipeline-builder/dist/pipeline.css";
+
+.svg {
+  overflow: hidden;
+}
+
+.fullscreen-svg-element {
+  width: 100% !important;
```

Comment:
`!important` is not ideal. I know you struggled to get it working otherwise, so maybe it's OK, but ideally these would not be necessary.

Reply:
That's a good point, but I'm not sure how to address the issue of inline styling without `!important`. The div's dimensions are getting overridden by inline style

Reply:
OK, we can always revisit if it becomes an issue.